src/rad/resources/schemas/l1_detector_guidewindow-1.0.0.yaml

```
@@ -0,0 +1,1408 @@
+            ndim: 2
+      edge_acq_centroid_errs:
+        title: Edge Acquisition Centroid Errors
+        description: |
+          2D array containing the 2nd moment errors of the GW edge acquisition difference
+          centroids (XX, YY, XY) (only used with WSM). This quantity is set to None for Dark
+          exposures when no centroids are taken.
+        anyOf:
+          - type: "null"
+          - tag: tag:stsci.edu:asdf/core/ndarray-1.*
+            datatype: float32
+            exact_datatype: true
+            ndim: 2
+      edge_acq_centroid_quality:
+        title: Edge Acquisition Centroid Quality
+        description: |
+            1D array containing strings that describe the quality of the edge acquisition
+            centroids (only used with WSM). For further descriptions of the, please consult
+            the Roman Documentation page. This quantity is set to None for Dark exposures
+            when no centroids are taken.
+        type: array
+        items:
+          anyOf:
+            - type: "null"
+            - type: string
+      edge_acq_centroid_times:
+        title: Edge Acquisition Centroid Times
+        description: |
+          1D array of times associated with the centroid of each edge acquisition resultant
+          difference (only used with WSM). The centroid times should lie between the pedestal
+          and signal resultant times. This quantity is set to None for Dark exposures when
+          no centroids are taken.
+        type: array
+        items:
+          anyOf:
+            - type: "null"
+            - tag: tag:stsci.edu:asdf/time/time-1.*
+
+  acq_data:
+    title: Acquisition Guide Window Data
+    description: |
+      All data received from the acquisition guide window data packets
+    type: object
+    properties:
+      pedestal_resultants:
+        title: Pedestal Resultants (DN)
+        description: |
+          3D array containing every pedestal resultant's pixel values.
+        tag: tag:stsci.edu:asdf/core/ndarray-1.*
+        ndim: 3
+        datatype: uint16
+        unit: "DN"
+        exact_datatype: true
+      signal_resultants:
+        title: Signal Resultants (DN)
+        description: |
+          3D array containing every signal resultant's pixel values.
+        tag: tag:stsci.edu:asdf/core/ndarray-1.*
+        ndim: 3
+        datatype: uint16
+        unit: "DN"
+        exact_datatype: true
+      lower_left_corners:
+        title: Lower Left Corners
+        description: |
+          2D array containing all the lower left corners of each guide
+          window resultant pair in science coordinates (x_start, y_start).
+        ndim: 2
+        datatype: uint16
+        exact_datatype: true
+      upper_right_corners:
+        title: Upper Right Corners
+        description: |
+          2D array containing all the upper right corners of each guide
+          window resultant pair in science coordinates (x_stop, y_stop).
+        ndim: 2
```

```suggestion
        tag: tag:stsci.edu:asdf/core/ndarray-1.*
        ndim: 2
```
Missing `tag` keyword.